modules/configuring-private-storage-endpoint-azure-user-provided-vnet-subnet.adoc

```diff
@@ -0,0 +1,101 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/configuring-private-cluster.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-private-storage-endpoint-azure-user-provided-vnet-subnet"]
+= Configuring a private storage endpoint on Azure with user-provided VNet and subnet names
+
+Use the following procedure to configure a storage account that has public network access disabled and is exposed behind a private endpoint on Azure.
+
+.Prerequisites
+
+* You have configured an image registry to run on Azure
+* You must know the VNet and subnet names used for your Azure environment.
```

Comment:
I forgot to mention users may also have a `networkResourceGroupName` - this was introduced to the api after the first round of PRs merged, so it wasn't present in the initial testing steps I provided.
```suggestion
* You must know the VNet and subnet names used for your Azure environment.
* If your network was configured in a separate resource group in Azure, you must also know it's name.
```